keyboards/handwired/orbweaver/orbweaver.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 a_marmot
```

Comment:
This file should be deleted.

Reply:
I am not a c programmer, but don't all c source codes need a header, even if empty?

Reply:
No, this file is essentially generated internally using the data in `info.json` file.

Reply:
Do I need a #include quantum.h somewhere else?

Reply:
Nope.

Reply:
It will not compile without orbweaver.h

I tried to move #include "quantum.h" to orbweaver.c, and remove reference to orbweaver.h.  Here's the error: 

./.build/obj_orbweaver/src/default_keyboard.h:39:10: fatal error: orbweaver.h: No such file or directory
 #include "orbweaver.h"
          ^~~~~~~~~~~~~
compilation terminated.

Reply:
Using `#include QMK_KEYBOARD_H` instead of `#include "orbweaver.h"` should work.

Also run `qmk clean` before compiling to make sure that any old generated files get removed properly.

Reply:
YES